src/panel_gwalker/_gwalker.py

```diff
@@ -84,17 +85,25 @@ class GraphicWalker(ReactComponent):
         Servable `GraphicWalker` object that creates a UI for visual exploration of the input DataFrame.
     """
 
-    object = param.DataFrame(doc="""The data to explore.""")
-    fields = param.List(doc="""Optional fields, i.e. columns, specification.""")
-    appearance = param.Selector(
-        default=None,
-        objects=[None, "media", "dark", "light"],
-        doc="""Dark mode preference: 'media', 'dark', 'light' or None.
-        If None the the appearance is derived from pn.config.theme""",
+    object: pd.DataFrame = param.DataFrame(
```

Comment:
You can see in this video that when we change the `object` data then the existing (chart)s stays around but have a strange behaviour. I think this is a bug in Graphic Walker or something we should have by deleting the existing charts if the `object` changes.

Could not figure out how to do it. Not important now.

https://github.com/user-attachments/assets/25d5445a-2386-4a92-a519-4c7d2937f90d